ppr-ui/src/views/LengthTrust.vue

```diff
@@ -16,16 +16,24 @@
             </v-row>
             <stepper class="mt-4" />
             <v-row no-gutters class="pt-10">
-              <v-col cols="auto" class="sub-header">
+              <v-col cols="auto" class="sub-header" v-if="registrationType !== 'RL'">
```

Comment:
enum for this file too here and below (just adding these comments to make it easier to check them off)

Reply:
Updated in latest commit to use enum.